### PR TITLE
muted ad issue

### PR DESCRIPTION
**Pull request by yaniv-li**
fixed ad shows as not muted when muted

### CODE REVIEW

File: src/plugins/ima/index.js [hunk 1]
```diff
@@ -264,6 +264,10 @@
         this.settings.postrollTimeout);
 
       this.adsLoader.requestAds(adsRequest);
+      if (this.player.muted()) {
```
Comment (mikeys) [inline src/plugins/ima/index.js]: `setAdMuted(this.player.muted())`

File: src/plugins/ima/index.js [hunk 2]
```diff
@@ -1035,6 +1023,25 @@
...
+        removeClass_(this.muteDiv, 'ima-non-muted');
+        this.adsManager.setVolume(0);
+        // Bubble down to content player
+        this.player.muted(true);
+        this.adMuted = true;
+        this.sliderLevelDiv.style.width = "0%";
+      } else {
+        addClass_(this.muteDiv, 'ima-non-muted');
+        removeClass_(this.muteDiv, 'ima-muted');
+        this.adsManager.setVolume(1);
```
Comment (mikeys) [inline src/plugins/ima/index.js]: Shouldn't you use the current volume of the player rather then set volume to 100%?

File: src/plugins/ima/index.js [hunk 3]
```diff
@@ -1035,6 +1020,25 @@
...
+        this.adMuted = true;
+        this.sliderLevelDiv.style.width = "0%";
+      } else {
+        addClass_(this.muteDiv, 'ima-non-muted');
+        removeClass_(this.muteDiv, 'ima-muted');
+        this.adsManager.setVolume(this.player.volume());
+        // Bubble down to content player
+        this.player.muted(false);
+        this.adMuted = false;
+        this.sliderLevelDiv.style.width = this.player.volume() * 100 + "%";
```
Comment (mikeys) [inline src/plugins/ima/index.js]: I assume this line needs to be changed as well?

Comment (yaniv-li) [reply src/plugins/ima/index.js]: way? this your code I just refactored it

Comment (mikeys) [reply src/plugins/ima/index.js]: @yaniv-li It doesn't mean that it's 100% correct :)

Comment (mikeys) [reply src/plugins/ima/index.js]: True, my bad.

**Pull request merged**